src/main/include/GeneralConstants.h

```diff
@@ -4,6 +4,56 @@
 
 #pragma once
 
+#include <cstddef>
+
+#include "Util/thirdparty/simplevectors.hpp"
+
+namespace vec = svector;
+
 namespace GeneralConstants{
   
+}
+
+namespace FieldConstants {
+  // positions of tags, in m
+  const vec::Vector2D TAG1 = {15.513558, 1.071626};
+  const vec::Vector2D TAG2 = {15.513558, 2.748026};
+  const vec::Vector2D TAG3 = {15.513558, 4.424426};
+  const vec::Vector2D TAG4 = {16.178794, 6.749796};
+  const vec::Vector2D TAG5 = {0.36195, 6.749796};
+  const vec::Vector2D TAG6 = {1.02743, 4.424426};
+  const vec::Vector2D TAG7 = {1.02743, 2.748026};
+  const vec::Vector2D TAG8 = {1.02743, 1.071626};
+
```

Comment:
might want to add an array of the tag positions just vector[8] TAGS = {TAG1, TAG2, TAG3...}

Reply:
ok yeah saw comment in odometry, do that